.gitignore

```
@@ -0,0 +1,3 @@
+.stack-work/
+public-api.cabal
```

I normally write this as `*.cabal` so it doesn't depend on the exact name of the package.
```suggestion
*.cabal
```